doc/develop/application/index.rst

```diff
@@ -431,7 +431,7 @@ should know about.
 
 * :makevar:`SHIELD`: see :ref:`shields`
 
-* :makevar:`ZEPHYR_MODULES`: A CMake list containing absolute paths of
+* :makevar:`ZEPHYR_MODULES`: A `CMake list`_ containing absolute paths of
```

Comment:
This seems like a stray change? Approved already but noticed after

Reply:
it's an external link

https://github.com/zephyrproject-rtos/zephyr/blob/d670f1ba6ecd7988c4220025513f55895c40f6c4/doc/develop/application/index.rst?plain=1#L1291

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links